docs/_admin/google.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: Configuring Google Maps
+has_children: false
+nav_order: 4
+---
+
+#Change the suggestions provider
+
+On the Dashboard, at the bottom section you can choose a different suggestions provider. Different providers work better depending on the region you are.
```

Comment:
It's not on the Dashboard actually, it's on the settings page. If there's an existing page of docs about this page, it should be linked here.